assets/Gruntfile.js

```diff
@@ -212,7 +212,8 @@ module.exports = function (grunt) {
           'bower_components/messenger/build/css/messenger-theme-block.css',
           'bower_components/messenger/build/css/messenger-theme-air.css',
           'bower_components/messenger/build/css/messenger-theme-ice.css',
-          'bower_components/messenger/build/js/messenger-theme-future.js'
+          'bower_components/messenger/build/js/messenger-theme-future.js',
+          'bower_components/fontawesome/css/font-awesome.css'
```

Comment:
[merge]
On Apr 12, 2016 4:50 PM, "Sam Padgett" <notifications@github.com> wrote:

> In assets/Gruntfile.js
> <https://github.com/openshift/origin/pull/8488#discussion_r59451065>:
>
> > @@ -212,7 +212,8 @@ module.exports = function (grunt) {
> >            'bower_components/messenger/build/css/messenger-theme-block.css',
> >            'bower_components/messenger/build/css/messenger-theme-air.css',
> >            'bower_components/messenger/build/css/messenger-theme-ice.css',
> > -          'bower_components/messenger/build/js/messenger-theme-future.js'
> > +          'bower_components/messenger/build/js/messenger-theme-future.js',
> > +          'bower_components/fontawesome/css/font-awesome.css'
>
> ├─┬ kubernetes-label-selector#0.0.18
> │ ├── angular#1.3.8 (1.3.21-build.156+sha.63c83ff available, latest is 1.5.4-build.4742+sha.709dc1c)
> │ ├── fontawesome#4.3.0 (latest is 4.6.1)
> │ └── selectize#0.11.2 (latest is 0.12.1)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8488/files/9a65c34e5857c902f89e715d08f9d8be25dbdccc#r59451065>
>